how-to-guides/07-cross-validation.ipynb

```diff
@@ -0,0 +1,1931 @@
+      "metadata": {
+        "id": "Wfk9Q7fIDHhI"
+      },
+      "source": [
+        "num_folds = 3\n",
+        "splits = KFold(n_splits=num_folds,shuffle=True,random_state=42)"
+      ],
+      "execution_count": 5,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "706JNk1gffbS"
+      },
+      "source": [
+        "We are going to train the model using all the folds we created. Every fold will train for 3 epochs, each of them with its metrics."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "mophHZUkICKI",
+        "outputId": "994f52c0-6ae6-47ac-bd21-73f7cea16127"
+      },
+      "source": [
+        "results_per_fold = []\n",
+        "\n",
+        "for fold_idx, (train_idx,val_idx) in enumerate(splits.split(np.arange(len(dataset)))):\n",
+        "\n",
+        "    print('Fold {}'.format(fold_idx + 1))\n",
+        "\n",
+        "    train_loader, val_loader = setup_dataflow(dataset, train_idx, val_idx)\n",
+        "    train_results, val_results = train_model(train_loader, val_loader)\n",
+        "    results_per_fold.append([train_results, val_results])"
+      ],
+      "execution_count": null,
+      "outputs": [
+        {
+          "metadata": {
+            "tags": null
+          },
+          "name": "stdout",
+          "output_type": "stream",
+          "text": [
+            "Fold 1\n",
+            "Training Results - Epoch[1] Avg accuracy: 0.73 Avg loss: 1.38\n",
+            "Training Results - Epoch[2] Avg accuracy: 0.84 Avg loss: 0.90\n",
+            "Training Results - Epoch[3] Avg accuracy: 0.89 Avg loss: 0.61\n",
+            "Fold 2\n",
+            "Training Results - Epoch[1] Avg accuracy: 0.74 Avg loss: 1.35\n",
+            "Training Results - Epoch[2] Avg accuracy: 0.85 Avg loss: 0.86\n"
+          ]
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "zv_i4JlklmyD"
+      },
+      "source": [
+        "#### Evaluation"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "sY6-lqKpfj6R"
+      },
+      "source": [
+        "After training the model, it is possible to evaluate its overall performance by averaging the validation accuracy of every fold."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "Ah5UoZz-zkoL",
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "outputId": "f222917d-176c-4a3c-a56d-bf84bb872a21"
+      },
+      "source": [
+        "acc_sum = 0\n",
+        "for n_fold in range(len(results_per_fold)):\n",
+        "  current_fold = results_per_fold[n_fold]\n",
+        "  print(f\"Validation Results - Epoch[{n_fold + 1}] Avg accuracy: {current_fold[1][2]['Accuracy']:.2f} Avg loss: {current_fold[1][2]['Loss']:.2f}\")\n",
```

Comment:
`current_fold[1][2]["Accuracy"]` means getting the Accuracy score of the validation ([1]) at epoch 3 ([2]). Maybe it worth mention that.